__tests__/__snapshots__/ord.test.js.snap

```diff
@@ -173,13 +173,13 @@ exports[`Tests for default ORD document Successfully create ORD Documents with d
   "openResourceDiscovery": "1.9",
   "packages": [
     {
-      "description": "Description for capjs ord",
+      "description": "Description for cap js ord",
```

Comment:
You are using ord sample project in xmpl project? 
@capire/ord-sample ?

Reply:
No, in the test directory we added a bookshop project which we load before running the test.